reference/orders.v3.yml

```diff
@@ -2365,6 +2402,9 @@ definitions:
       total_tax:
         type: number
         description: 'Total tax amount refunded back to shopper. Note: order_level_amount does not affect tax liability. This can be negative amount indicating we have collected tax by virtue of refunding less to the customer'
```

Comment:
```suggestion
        description: 'Total tax amount refunded back to the shopper. Note: `order_level_amount` does not affect tax liability. This can be a negative amount indicating we have collected tax by refunding less to the customer.'
```